agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1805,24 +1805,17 @@ bool backhaul_manager::handle_1905_autoconfiguration_response(ieee1905_1::CmduMe
 bool backhaul_manager::send_slaves_enable()
 {
     auto iface_hal = get_wireless_hal();
-    iface_hal->refresh_radio_info();
-    const auto &radio_info = iface_hal->get_radio_info();
 
     for (auto soc : slaves_sockets) {
         auto notification =
             message_com::create_vs_message<beerocks_message::cACTION_BACKHAUL_ENABLE_APS_REQUEST>(
                 cmdu_tx);
 
         if (soc->sta_iface == m_sConfig.wireless_iface) {
-            notification->channel()   = iface_hal->get_channel();
-            notification->bandwidth() = radio_info.bandwidth;
-            notification->center_channel() =
```

Comment:
I think I've asked this question before, but I don't remember the answer:

This will leave the bandwidth and center_channel uninitialized, so they will use whatever was in the buffer before. Why is that acceptable? (Note that we already are in that situation if son->sta_iface != wireless_iface...)

Reply:
You are right, you did ask.

Currently, the refresh_radio_info doesn't support STAs, and thus the BW and center_channel for STAs.
I can set it to zeros (although not a valid value) until we add this functionality to the refresh_radio_info